File: Math, Logical Implememt/GCD, LCM/acmicpc.net/9613/Main.js
"use strict";

const inputFilePath = "/sample.txt";
const input = require("fs")
  .readFileSync(__dirname + inputFilePath)
  .toString()
  .trim()
  .split("\n");
// const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");
const n = Number(input[0]);
let gcd = (a, b) => {
  while (b !== 0) {
    let tmpA = a;
    let tmpB = b;
    a = tmpB;
    b = tmpA % tmpB;
  }
  return a;
};
for (let i = 1; i <= n; i++) {
  const arr = input[i].split(" ").map((x) => Number(x));
  let ans = 0;
  for (let i = 1; i <= arr[0]; i++) {
    for (let j = i + 1; j <= arr[0]; j++) {
      ans += gcd(arr[i], arr[j]);
    }
  }
  console.log(ans);
}

File: Math, Logical Implememt/GCD, LCM/acmicpc.net/9613/Main.py
# import sys
# input = sys.stdin.readline
file = open("./01 ing/acmicpc.net/sample.txt", "r")
input = file.readline
n = int(input())

def gcd(a,b):
  while(b!=0):
    tmpA = a;
    a = tmpB;
    tmpB = b;
    b = tmpA%tmpB;
  return a;

for _ in range(n):
  arr = list(map(int,input().split()))
  ans = 0;
  for i in range(1, len(arr)):
    for j in range(i+1,len(arr)):
      ans+=gcd(arr[i], arr[j]);
  print(ans)